### PR TITLE
PA 2744 Add building/land name to top of view/edit

### PR DESCRIPTION
**Pull request by agalbrai**
image
image
image

### CODE REVIEW

File: frontend/src/features/mapSideBar/SidebarContents/BuildingForm.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ const FormContentWrapper = styled.div`
   flex-direction: column;
   width: 100%;
   height: 100%;
-  margin-bottom: 50px;
+  margin-bottom: 70px;
```
Comment (agalbrai) [inline frontend/src/features/mapSideBar/SidebarContents/BuildingForm.tsx:60]: bottom strip was being cutoff

**Review by jdtoombs — APPROVED** [review]

File: frontend/src/features/mapSideBar/components/MapSideBarLayout.tsx [hunk 2]
```diff
@@ -44,6 +53,18 @@ const Title = styled.span`
   text-align: left;
 `;
 
+/** Shortens property name to a reasonable length as some are quite long */
+const formattedName = (name?: string) => {
+  if (name) {
```
Comment (Fosol) [inline frontend/src/features/mapSideBar/components/MapSideBarLayout.tsx:58]: Can you just do `return name?.substring(0, 80)`?

**Review by Fosol — APPROVED** [review]

File: frontend/src/features/mapSideBar/components/MapSideBarLayout.tsx (hunk 2 above)
Comment (devinleighsmith) [reply frontend/src/features/mapSideBar/components/MapSideBarLayout.tsx:58]: created an AbbreviatedText component last sprint for this purpose.

**Comment by devinleighsmith** [comment]
If snyk is giving you problems, you may need to force push to retrigger the check (changing the commit message, for example).

**Review by devinleighsmith — APPROVED** [review]

File: frontend/src/features/mapSideBar/components/MapSideBarLayout.tsx (hunk 2 above)
Comment (agalbrai) [reply frontend/src/features/mapSideBar/components/MapSideBarLayout.tsx:58]: oh shoot I'll use that thanks!

**Pull request merged**